presto-main/src/main/java/com/facebook/presto/metadata/MetadataListing.java

```diff
@@ -73,6 +74,18 @@ private MetadataListing() {}
         return accessControl.filterTables(session.getRequiredTransactionId(), session.getIdentity(), prefix.getCatalogName(), tableNames);
     }
 
+    public static Set<GrantInfo> listTablePrivileges(Session session, Metadata metadata, AccessControl accessControl, QualifiedTablePrefix prefix, String grantee)
+    {
+/*        Set<PrivilegeInfo> grants = metadata.listTablePrivileges(session, prefix, grantee).stream()
```

Comment:
You completely lose the 
 .filter(grantInfo -> grantInfo.getIdentity().getUser().equalsIgnoreCase(grantee))
Also, remove this comment.